src/components/ExtraDataRightMenu/ExtraDataMenu.js

```diff
@@ -78,15 +78,18 @@ function ExtraDataMenu(props) {
 		}))
 	}
 
-	/* Don't load this bar if there are no dataset options */
+	/* Don't load this bar if there are no dataset options */  
 	return (radioData && selectedCounty) ? (
-		<div className="extraData-wrapper">
+    <div className="extraData-wrapper">
+      {console.log("This is what was to the radio buttons before, ", Object.keys(radioData))}
```

Comment:
Do we need this console statement or is it a left over from building/debugging things?

Reply:
Thanks for the catch. Got it removed.